### PR TITLE
feat(test): Add UT for the util packages

### LINKED ISSUE #49: Unit testing for all packages
### Potential Scope

-  Add unit testing for all the packages and the new packages to added.
-  Codecoverage integration

### Packages
- [x] pkg/blockdevice
- [ ] #67
- [x] pkg/persistentvolumeclaim
- [x] pkg/storage
- [x] pkg/util
- [x] #63

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
### What does this PR do?
- This PR adds the Unit Tests for the util packages.
- Removed unused code snippets.

### Closed #49

### CODE REVIEW

File: pkg/util/k8s_utils_test.go [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			"Valid values",
+			args{cvrList: &cstorv1.CStorVolumeReplicaList{Items: []cstorv1.CStorVolumeReplica{{Status: cstorv1.CStorVolumeReplicaStatus{
+				Phase:              "Init",
+				Capacity:           cstorv1.CStorVolumeReplicaCapacityDetails{Total: "5.0 GiB", Used: "2.5 GiB"},
```
Comment (vharsh) [inline pkg/util/k8s_utils_test.go]: Adding a different value for the CVR which is stuck in init, to show that this used capacity will not be counted
```suggestion
				Capacity:           cstorv1.CStorVolumeReplicaCapacityDetails{Total: "5.0 GiB", Used: "2.1 GiB"},
```
maybe add another healthy CVR with a Healthy status.Phase & a different size to show that the first Healthy CVR's used capacity is taken in this method.

File: pkg/util/testdata_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+package util
+
+import (
+	cstortypes "github.com/openebs/api/v2/pkg/apis/types"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
```
Comment (vharsh) [inline pkg/util/testdata_test.go]: (nit) Re-order imports, the standard libraries should come in the first line

File: pkg/util/k8s_utils_test.go [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+limitations under the License.
+*/
+
+package util
+
+import (
+	cstorv1 "github.com/openebs/api/v2/pkg/apis/cstor/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/storage/v1"
+	"testing"
```
Comment (vharsh) [inline pkg/util/k8s_utils_test.go]: re-order imports

**Review by vharsh — COMMENTED** [review]
lgtm, minor suggestions for re-order of imports, etc

File: pkg/util/k8s_utils_test.go (hunk 3 above)
Comment (Abhinandan-Purkait) [reply pkg/util/k8s_utils_test.go]: Done.

File: pkg/util/testdata_test.go (hunk 2 above)
Comment (Abhinandan-Purkait) [reply pkg/util/testdata_test.go]: Done.

**Review by vharsh — APPROVED** [review]
lgtm

**Pull request merged**